### PR TITLE
some Spellcheck/fixes discoverd by lift

### PR DESCRIPTION
**Pull request by pheiduck**
continue: #8616

### CODE REVIEW

File: lib/firefox-db2pem.sh [hunk 1]
```diff
@@ -24,14 +24,14 @@
 # It extracts all ca certs it finds in the local Firefox database and converts
 # them all into PEM format.
 #
-db=`ls -1d $HOME/.mozilla/firefox/*default*`
+db="ls -1d $HOME/.mozilla/firefox/*default*"
```
Comment (vszakats) [inline lib/firefox-db2pem.sh:27]: Should this be?:
```suggestion
db=$(ls -1d $HOME/.mozilla/firefox/*default*)
```

**Comment by vszakats** [comment]
Here's is the `shellcheck` output for this script (mostly double-quotes missing around `$()`):
```
In firefox-db2pem.sh line 27:
db=`ls -1d $HOME/.mozilla/firefox/*default*`
   ^-- SC2006 (style): Use $(...) notation instead of legacy backticks `...`.
           ^---^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
db=$(ls -1d "$HOME"/.mozilla/firefox/*default*)


In firefox-db2pem.sh line 34:
currentdate=`date`
            ^----^ SC2006 (style): Use $(...) notation instead of legacy backticks `...`.

Did you mean: 
currentdate=$(date)


In firefox-db2pem.sh line 46:
certutil -L -h 'Builtin Object Token' -d $db | \
                                         ^-^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
certutil -L -h 'Builtin Object Token' -d "$db" | \


In firefox-db2pem.sh line 50:
while read nickname; \
      ^--^ SC2162 (info): read without -r will mangle backslashes.


In firefox-db2pem.sh line 51:
 do echo $nickname | sed -e "s/Builtin Object Token://g"; \
         ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
 do echo "$nickname" | sed -e "s/Builtin Object Token://g"; \


In firefox-db2pem.sh line 52:
eval certutil -d $db -L -n "$nickname" -a ; \
                 ^-^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
eval certutil -d "$db" -L -n "$nickname" -a ; \

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2162 -- read without -r will mangle backs...
  https://www.shellcheck.net/wiki/SC2006 -- Use $(...) notation instead of le...
```

**Review by bagder — APPROVED** [review]

**Comment by bagder** [comment]
Thanks!

**Comment by bagder** [comment]
Oops I referred to the wrong PR in the commit... :cry:

**Pull request closed without merging**